src/libostree/ostree-repo.h

```diff
@@ -735,7 +735,7 @@ ostree_repo_checkout_tree (OstreeRepo               *self,
                            GError                  **error);
 
 /**
- * OstreeRepoCheckoutOptions:
+ * OstreeRepoCheckoutOptions: (skip)
```

Comment:
also  `subpath` and `devino_to_csum_cache` are not valid, as their offset is not correct.  Only `mode` and `overwrite_mode` could be used correctly.